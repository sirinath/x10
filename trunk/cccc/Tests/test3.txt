# Dependency chain.

0.0 W a 1
1.0 R a 1
1.1 W b 1
2.0 R b 1
2.1 W c 1
3.0 R c 1
3.1 W d 1
4.0 R d 1
4.1 R a 0

0.0 -> 1.0
1.1 -> 2.0
2.1 -> 3.0
3.1 -> 4.0
4.1 -> 0.0
