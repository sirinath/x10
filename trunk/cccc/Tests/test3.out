0.0 w a 1
1.0 r a 1
1.1 w b 1
2.0 r b 1
2.1 w c 1
3.0 r c 1
3.1 w d 1
4.0 r d 1
4.1 r a 0
# Manually specified dependencies
0.0 -> 1.0
1.1 -> 2.0
2.1 -> 3.0
3.1 -> 4.0
4.1 -> 0.0
# Program-order dependencies
1.0 -> 1.1
2.0 -> 2.1
3.0 -> 3.1
4.0 -> 4.1
# Propagated dependencies
0.0 -> 1.1
1.0 -> 2.0
1.0 -> 2.1
1.1 -> 2.1
2.0 -> 3.0
2.0 -> 3.1
2.1 -> 3.1
3.0 -> 4.0
3.0 -> 4.1
3.1 -> 4.1
4.0 -> 0.0
Happens-before dependencies for task 0
0.0 -> 1.0
0.0 -> 1.1
1.0 -> 1.1
2.0 -> 2.1
3.0 -> 3.1
4.0 -> 0.0
4.0 -> 4.1
4.1 -> 0.0
Happens-before dependencies for task 1
0.0 -> 1.0
0.0 -> 1.1
1.0 -> 1.1
1.0 -> 2.0
1.0 -> 2.1
1.1 -> 2.0
1.1 -> 2.1
2.0 -> 2.1
3.0 -> 3.1
4.0 -> 4.1
Happens-before dependencies for task 2
1.0 -> 1.1
1.0 -> 2.0
1.0 -> 2.1
1.1 -> 2.0
1.1 -> 2.1
2.0 -> 2.1
2.0 -> 3.0
2.0 -> 3.1
2.1 -> 3.0
2.1 -> 3.1
3.0 -> 3.1
4.0 -> 4.1
Happens-before dependencies for task 3
1.0 -> 1.1
2.0 -> 2.1
2.0 -> 3.0
2.0 -> 3.1
2.1 -> 3.0
2.1 -> 3.1
3.0 -> 3.1
3.0 -> 4.0
3.0 -> 4.1
3.1 -> 4.0
3.1 -> 4.1
4.0 -> 4.1
Happens-before dependencies for task 4
1.0 -> 1.1
2.0 -> 2.1
3.0 -> 3.1
3.0 -> 4.0
3.0 -> 4.1
3.1 -> 4.0
3.1 -> 4.1
4.0 -> 0.0
4.0 -> 4.1
4.1 -> 0.0
No cycles found: plausible CCCC execution
